### DISCUSSION TITLE
Auto-adjust dashboard view size between phones and computers

### DISCUSSION THREAD

**Discussion by pyeh-seg** [grafana/grafana#62535] [Dashboards, Panels & Visualizations]
Is there a way to enable auto-adjust screen size? Such that when we open Grafana on our phone, it would adjust to the screen size?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
